tests/ti_deps/deps/test_trigger_rule_dep.py

```diff
@@ -163,6 +163,26 @@ def test_one_failure_tr_success(self, get_task_instance):
         )
         assert len(dep_statuses) == 0
 
+    def test_one_done_tr_success(self, get_task_instance):
```

Comment:
Can we have more tests to cover the full functionality of the trigger rule?
We have here 1 test for success..
We need also 1 for skip, 1 for upstream failed I think?

Reply:
Added